docs/cimpl6LanguageReference.md

```diff
@@ -1057,124 +1044,127 @@ With FHIR as the target output, the right-hand side of the map statement can poi
 
-##### Discriminator Type
+#### Discriminator Type: `slice on type`
 
-The discriminator type defines how a slice is differentiated from another slice for the same `Property`. By default, CIMPL uses the FHIR **value** discriminator type. However, a different discriminator type can be declared using `slice on type` followed by one of the following FHIR-defined discriminator types:
+The discriminator type defines how the discriminator is used to differentiate instances from one another. By default, CIMPL uses the FHIR **value** discriminator type. However, a different discriminator type can be declared using `slice on type` followed by one of the following FHIR-defined discriminator types:
 
-| `Slice on` Type | Definition |
+| `slice on type` | Interpretation |
 | ------------- | ---------- |
-| **value**| The slices have different values in the discriminator |
-| **exists** | The slices are differentiated by the presence or absence of the `sliced on`|
-| **pattern** |The slices have different values in the discriminator, as determined by testing them against the applicable **ElementDefinition.pattern[x]**|
-| **type** |The slices are differentiated by data type of the discriminator to a specified **profile**|
-|**profile** |Not recommended. The slices are differentiated by conformance of the discriminator to a specified **profile**. This can only be used when the discriminator is an `Entry`.|
+| `value`| The slices have different values in the discriminator |
+| `exists` | The slices are differentiated by the presence or absence of the discriminator |
+| `pattern` |The slices have different values in the discriminator, as determined by testing them against the applicable **ElementDefinition.pattern[x]** |
+| `type` |The slices are differentiated by the data type of the discriminator |
+| `profile` | The slices are differentiated by conformance of the discriminator to a specified **profile**. This can only be used when the discriminator is an `Entry`, and is generally not recommended for performance reasons. |
 
 #### Slice Strategy
 <!-- Carmela - confirm underscore with slice strategy check the code -->
 
-The `slice strategy` indicates how the CIMPL `Property` should map to slices in the target FHIR **element**. The default `slice strategy` adds a single slice, corresponding to the CIMPL data type, in the target FHIR array.  To map a CIMPL `Property` with `includes` constraints, such that each included CIMPL data type becomes a slice in the target FHIR array, specify _`slice strategy =` **includes**_.
+The `slice strategy` indicates how the CIMPL `Property` should map to slices in the target FHIR **element**. The default `slice strategy` adds a single slice, corresponding to the CIMPL data type, in the target FHIR array.  To map a CIMPL `Property` with `includes` constraints, such that each included CIMPL data type becomes a slice in the target FHIR array, specify `slice strategy = includes`.
```

Comment:
same comment as above for **includes**

Reply:
`slice strategy = includes` is exactly how it appears in the map file. So, no bold.